ksp_plugin_test/vessel_test.cpp

```diff
@@ -39,18 +39,33 @@ class VesselTest : public testing::Test {
 
 using VesselDeathTest = VesselTest;
 
+TEST_F(VesselDeathTest, Uninitialized) {
+  EXPECT_DEATH(vessel_->history();, "is_synchronized");
```

Comment:
`();},` (user opens mouth, winks, and wears a plumed hat) would have pleased the linter.
Sadly it turns out `(),` is enough.